phlop/timing/scope_timer.py

```
@@ -132,29 +134,46 @@
         kinder(total, root, 0, total)
 
 
-def write_root_as_csv(scope_timer_file, outfile, headers=None, regex=None):
+def write_variance_across(scope_timer_file_glob, outfile):
     from contextlib import redirect_stdout
 
     with open(outfile, "w") as f:
         with redirect_stdout(f):
```

Ruff

Use a single `with` statement with multiple contexts instead of nested `with` statements

Combine `with` statements

(SIM117)